src/xcode/ENA/ENA/Source/Services/CoronaTestService/__tests__/MockCoronaTestService.swift

```diff
@@ -17,15 +17,15 @@ class MockCoronaTestService: CoronaTestServiceProviding {
 
```

Comment:
Should this file not be renamed to `MockUserCoronaTestService`?

Reply:
I did not prefix the `CoronaTestService` of the protocol with `User` as that would have led to many more variables to rename and I already thought there were so many 😅 Not sure currently if we still should rename it, we can discuss it. But if we do, it for sure should be a different PR 😄

Reply:
You are right, this would lead into more changes so a separate PR would make sense.